src/test/core/completionGenerator.test.ts

```diff
@@ -32,19 +32,31 @@ suite("Completion generation tests", () => {
             modelsParams: createPredefinedProofsModelsParams(predefinedProofs),
             services: createDefaultServices(),
         };
+        const abortController = new AbortController();
         try {
-            return await Promise.all(
+            await environment.coqLspClient.openTextDocument(
```

Comment:
Shouldn't the code below be wrapped in the `try-finally` guard? (also relevant to the other `open`/`close` calls in the other tests)